canonical-webteam/snapstore-module
# Create skeleton to prepare project

# Summary

Fixes https://github.com/canonicalltd/snap-squad/issues/528
Create skeleton for project

# QA

Nothing to QA mostly code generated to prepare project.
Make sure I haven't forgot anything (you can check on this project https://github.com/canonical-webteam/snapstore-api )
